spec/unit/recipes/mac_os_x_client_spec.rb

```diff
@@ -1,18 +1,25 @@
 require 'spec_helper'
 
 describe 'ntp::mac_os_x_client' do
-  let(:chef_run) { ChefSpec::SoloRunner.new(platform: 'mac_os_x', version: '10.11.1').converge('ntp::mac_os_x_client') }
+  let(:chef_run) do
+    ChefSpec::SoloRunner.new(
+      platform: 'mac_os_x',
+      version: '10.11.1'
```

Comment:
This is going to throw a deprecation warning on the new fauxhai as 10.11.1 mock was deprecated. Use 10.12 instead